chips/veer_el2/src/lib.rs

```diff
@@ -1,13 +1,14 @@
 // Licensed under the Apache License, Version 2.0 or the MIT License.
 // SPDX-License-Identifier: Apache-2.0 OR MIT
 // Copyright Tock Contributors 2022.
-
-//! Drivers and chip support for SweRVolf.
+// Copyright (c) 2024 Antmicro <www.antmicro.com>
```

Comment:
I don't the renames justify this

Reply:
I want to repeat @alevy's comment from above:

> Our general policy is that people are free to _add_ copyright information for non-trivial changes:
> 
> > Substantial updates to an artifact MAY add additional copyright notices to an
> > artifact. In general, modifications to a file are expected to retain existing
> > copyright notices.

And also, from https://github.com/tock/tock/blob/89f51be81f50380fd7a409ee34a8dd3072131ebd/doc/reference/trd4-legal.md:

> Copyrights in Tock projects are retained by their contributors. No
> copyright assignment is required to contribute to Tock projects.

> For full authorship information, see the version control history.

However the policy is written, I wouldn't interpret the "Substantial updates to an artifact MAY [...]" to discourage contributors of adding these lines, especially when it helps to make a case for their employer, etc. My reading of this policy is to prevent small drive-by changes to also add these notices.